sites/geohub/src/lib/types/DatasetFeature.ts

```diff
@@ -23,6 +23,8 @@ export interface DatasetFeature {
 		tags?: Tag[];
 		no_stars?: number;
 		is_star?: boolean;
+		collection_id?: string;
+		product?: string;
```

Comment:
Is this `product` property necessary? I saw you are adding `product` tag too. I think using tag is enough. Could you remove `product` from properties?